allennlp/tests/commands/test_install_test.py

```diff
@@ -0,0 +1,12 @@
+# pylint: disable=invalid-name,no-self-use
+import os
+
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.commands.test_install import _get_module_root
+
+
+class TestTestInstall(AllenNlpTestCase):
```

Comment:
What is the motivation for this test?  A brief comment would be helpful.  I'm confused because it's a test, in the tests folder, that's checking to see whether the tests folder exists.

Reply:
It looks like this is a rename rather than a new file.  While that doesn't change my confusion, you don't need to address it as part of this PR.

Reply:
I agree that this is quite confusing. The context is that when a user runs `test-install`, we have no idea where they're running it from, so we do an `os.chdir` to the module root in order to get all the paths to resolve properly.

The logic within `test-install` is pretty hard to test in its entirety ([see discussion here](https://github.com/allenai/allennlp/pull/1213#issuecomment-388897160)), so this test is basically a test for that `os.chdir` code, checking that we correctly find the path to `chdir` to.

does that help? I'll add this as a comment too.

Reply:
Yes that helps!